mypy/nodes.py

```diff
@@ -457,6 +476,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             arg.set_line(self.line)
         return self
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
```

Comment:
off the top of my head I can't remember how tokens -> argument node happens, it's possible they already have the column (and actually from editing the tests, my gut says they do)